manifests/templates/cdi-uploadproxy-route.yaml.in

```diff
@@ -0,0 +1,14 @@
+
+apiVersion: v1
+kind: Route
+metadata:
+  name: cdi-uploadproxy-route
+  namespace: {{ .Namespace }}
+spec:
+  # change to a valid FQDN in your organization
+  host: cdi-uploadproxy.example.com
```

Comment:
maybe comment this out as this should be optional in a properly configured openshift cluster.

Reply:
done